src/Foundation/NSString.cs

```diff
@@ -116,17 +116,33 @@ public static IntPtr CreateNative (string str)
 		}
 
 		public static IntPtr CreateNative (string str, bool autorelease)
+		{
+			return CreateNative (str, 0, str.Length, autorelease);
+		}
+
+		public static IntPtr CreateNative (string str, int start, int length)
```

Comment:
We try to avoid abbreviations for parameter names in public API, so `string value` instead of `string str` (yes, I know the existing overload is using `str`, but that's no reason we can't make new overloads better).